google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -310,7 +310,9 @@ def get_indexes(self, connection, table_name, schema=None, **kw):
 FROM information_schema.indexes as i
 JOIN information_schema.index_columns AS ic
     ON ic.index_name = i.index_name AND ic.table_name = i.table_name
-WHERE i.table_name="{table_name}"
+WHERE
+    i.table_name="{table_name}"
+    AND i.index_type != 'PRIMARY_KEY'
```

Comment:
Locally I've implemented drop table mechanism which first drops indexes. The problem is that it also tries to drop primary keys, which is not correct. Fixing it.

Reply:
This fixes the drop table mechanism that you've implemented. Is this method used elsewhere? Is this a method that users will call? If so, could you confirm that this method should not be returning primary key indexes?

Reply:
Well, it's not directly related to the drop table mechanism, the method is for introspection. Still, it's used by a bunch of methods, including the one dropping tables.

Users can use this method - they can instant an `Inspector()` object and use it for introspection. As primary key indexes are something controlled by Spanner itself, I assume we can avoid showing them to user, considering it as an obvious and auto regulated thing, which works without user's control.

Reply:
If other database products such as MySQL do not return primary key indexes then this LGTM :+1:  Please confirm that this is the case

Reply:
Yeah, here is a couple of examples

SQLite:
https://github.com/sqlalchemy/sqlalchemy/blob/76b33bb435f1f55111cd68cde9b7cb735c748bcd/lib/sqlalchemy/dialects/sqlite/base.py#L2454

MySQL:
https://github.com/sqlalchemy/sqlalchemy/blob/76b33bb435f1f55111cd68cde9b7cb735c748bcd/lib/sqlalchemy/dialects/mysql/base.py#L3170-L3171

PostgreSQL:
https://github.com/sqlalchemy/sqlalchemy/blob/3fec5028e695ad138aa46a0ae66c55e8bcb653f6/lib/sqlalchemy/dialects/postgresql/base.py#L4048